### PR TITLE
implement mdx remote

### PR DESCRIPTION
**Pull request by jescalan**
Implements [this RFC](https://docs.google.com/document/d/1Fk_TXUZzBiNIo4fGOhlL_v4kXyToLDgockc2fq0THmU/edit)! 

Preview the site here: https://nomad-3038bs301.vercel.app/

### CODE REVIEW

**Review by tgross — APPROVED** [review]
I spot-checked the preview (in particular anywhere there were content edits and places I know there are banners or other fancy bits). LGTM

File: website/data/intro-navigation.js [hunk 1]
```diff
@@ -10,6 +10,33 @@ export default [
   { category: 'vs', content: ['ecs', 'mesos', 'terraform'] },
   {
     category: 'getting-started',
-    content: ['running', 'jobs', 'cluster', 'ui', 'next-steps'],
+    name: 'Getting Started',
+    content: [
```
Comment (DingoEatingFuzz) [inline website/data/intro-navigation.js:14]: For what it's worth, the intro page is no longer linked to from anywhere (though it's still in search results).

**Review by DingoEatingFuzz — APPROVED** [review]
:shipit: :shipit: :shipit: :shipit: :shipit: :shipit: 

My only comment is of the fyi variety. I'm excited to see this merge!

File: website/data/intro-navigation.js (hunk 1 above)
Comment (jescalan) [reply website/data/intro-navigation.js:14]: Yeah, we can definitely remove it if the team feels like that's the right move. I just didn't feel like that was my choice to make 😅

**Pull request merged**